src/cheri/debug-integration.adoc

```diff
@@ -164,30 +164,44 @@ The <<dscratch1>> register is extended to hold a capability.
 .Debug scratch 1 capability register
 include::img/dscratch1creg.edn[]
 
-[#dinfc,reftext="dinfc"]
-==== Debug Infinite Capability Register (dinfc)
+[#drootcsel,reftext="drootcsel"]
+==== Debug Root Capability Selector (drootcsel)
 
-<<dinfc>> is a debug mode accessible capability CSR. The address and access details are shown in
+<<drootcsel>> is a debug mode accessible integer CSR.  The address and access details are shown in
 xref:default-csrnames-added[xrefstyle=short].
 
-The reset value is the <<infinite-cap>> capability.
+It selects which <<root-cap>> capability is exposed through <<drootc>>.
+The reset value is `0`, which must cause <<drootcsel>> to expose a
+<<root-rx-cap>> capability.
 
-If {cheri_default_ext_name} is implemented:
+If {cheri_default_ext_name} is implemented, this <<root-rx-cap>> is further
```

Comment:
I feel this text would be better as part of drootc

Reply:
OK.